internal/onprem/event_extractor.go

```diff
@@ -183,13 +183,41 @@ func (e *EventExtractor) extractEvents(tmpdir string, eventChannel chan types.Ev
 		},
 	}
 
+	versions, err := getVersionsFromFile(tmpdir, "/*/versions.json")
+	if err != nil {
+		e.logger.WithFields(logrus.Fields{
+			"err": err,
+		}).Warning("error extracting versions")
+	}
 	for _, eventTypeMatch := range eventTypesMatches {
-		e.extractEventsForEventType(eventTypeMatch, tmpdir, eventChannel)
+		e.extractEventsForEventType(eventTypeMatch, tmpdir, versions, eventChannel)
 	}
 	close(eventChannel)
 }
 
-func (e *EventExtractor) extractEventsForEventType(eventTypeMatch EventTypeMatch, tmpdir string, eventChannel chan types.EventEnvelope) {
+func getVersionsFromFile(tmpdir string, filename string) (map[string]string, error) {
+	versions := map[string]string{}
+	files, err := filepath.Glob(fmt.Sprintf("%s%s", tmpdir, filename))
```

Comment:
```suggestion
	files, err := filepath.Glob(tmpdir + filename)
```
nit: a bit more concise